Source/Core/InputCommon/ControllerEmu/Setting/NumericSetting.h

```diff
@@ -69,9 +83,11 @@ class NumericSettingBase
   const char* GetUIName() const;
   const char* GetUISuffix() const;
   const char* GetUIDescription() const;
+  const bool IsAdvanced() const;
 
 protected:
   NumericSettingDetails m_details;
+  std::function<void()> m_callback = nullptr;
```

Comment:
```suggestion
  std::function<void()> m_callback;
```

Is identical and a tad shorter, since std::function instances are empty by default

Reply:
> IMO, there should be a warning to users that changing the gate radius beyond default won't correspond to behavior of a retail controller, but that of a controller with a modified shell. The user can then ask for the warning to never show up again. This way there's no need to ceate an "Advanced" section.

One-time warnings are REALLY easy to forget in my experience. Even repeated warnings. I like having an advanced section since it retains all of the functionality, but it's physically different than the other controls and looks different (new window that says Advanced). It creates an easy spot to partition the settings in people's brains. That way they feel like they're deep in the underbelly of the program and understand it might mess things up. Advanced usually indicates great potential to ruin something in my experience.